swift/Glacier2/callback/Sources/Server/main.swift

```diff
@@ -38,8 +35,7 @@ func run() -> Int32 {
             return 1
         }
         let adapter = try communicator.createObjectAdapter("Callback.Server")
-        let cb = CallbackI()
-        try adapter.add(servant: cb, id: stringToIdentity("callback"))
+        try adapter.add(servant: CallbackI(), id: stringToIdentity("callback"))
```

Comment:
Need update to use the new `Disp` structs, 

`try adapter.add(servant: CallbackDisp(CallbackI()), id: stringToIdentity("callback"))`